### DISCUSSION TITLE
About S require no miss

### DISCUSSION THREAD

**Discussion by LocalGuard039** [ppy/osu#26742] [Ideas]
I believe everyone heard something like 99%A, yes? I would say full combo is impressive for S but a high accuracy is impressive too. And this no miss requirement just isn't feels right, How can this miss equal to roughly 5% accuracy?

I saw many people comment 95%acc as "easy to obtain", I wonder what playtime they have and how much gap between their comfort difficulty and their pass difficulty. For me with 2 days playtime, I'm struggle with 300 ur and barely see an A at 4.0* , my pass difficulty is 4*full, normally. It require too many time to complish along with mechanic that ok gives only 33%acc and meh give worse. I think propose like raise requirement to 98% is just too bad, 95% is fine as it is, without old players interfere this would be so much QoL for non-hardcore player so far.

_And there is a simple comparison between stable and lazer:

stable A is 90%great with miss, S is 90%great no miss(if you can hit 90%great, then 1%meh normally won't be an issue)
lazer is A 90%acc, S is 95%acc

So basically lazer just turned no miss gap into a 5%acc gap, and now with no miss then there is 2 gap between them, this isn't even a proper step back. Some may say S is around 95%acc no miss before, but this rank thing is just need to compare, A value will affect S value, those standardize thing must be done with determination. 
Or perhap if I say,is stable S require 95%great no miss legit to you? or lazer S require 90%acc no miss legit to you? that is, the two rank already have enough gap between them. If you see this 5%acc isn't a gap large enough, then you may play osu too much go touch some grass, and plus everyone of us eventually get "skill issued" while challenging higher difficulty map, your "95% is too easy opinion" doesn't stand ground_

PS:
Your S requirement change claims that you do this because you heard people. But I never heard there is a vote, I also see negative voice greatly. Do you vote underground, with mostly high skill player or something? I would guess you asked question like "Do you want fc requirement of no fc requirement?", such misleading question without giving a third option.
If no, then plus what you did to HP drain earlier, these shows you always use "step back" as your first move to solve issues and only consider more while problem caused by these happened, this is not good for this game to adapt the changing gaming enviroment nor do benefit to community.

**Comment by LocalGuard039** [comment]
Oh, one more thing. That S dropped to A animation is bad too. You can't make a accurate play looks like a joke because some miss, that should be a if miss then result screen show A at 90-100%accuracy , S and SS doesn't appear at all and if no miss, S and SS zone appear like hp replenish. So that conclude all, this from idea to design both didn't consider players, would be a bad update if things goes on.

**Comment by marioman63** [comment]
One other thing to note, is that osu is based on the DS rhythm game series "Osu! Tatakae! Ouendan" (or known in the west by its english entry Elite Beat Agents), where a S rank was reserved for a full combo above 90% accuracy. It seems clear people liked this aspect that has been a standard of this type of rhythm game since before osu! existed. I too think S should be a FC indicator as well. People have suggested adding a dedicated FC stamp on scores, but I think that just complicates the UI in the game's current design. S rank did double duty and it is common knowledge that S=FC, so by staying with the familiar, it is easy to attract returning players (new players it shouldn't matter to because they wouldn't know either method).

**Reply by LocalGuard039** [reply]
Well, that's when old players versus new players. But this tradition is kept for quite long enough, right? It's already too much time gap between before and now. And many osu stable players didn't play Elite Beat Agents, too. Now you have so many community made beatmaps, with various elements that push difficulty quickly, that the old DS would no longer compare. You see, if this is concered about returning player, they should return quite earlier instead of this 2024. And we don't have time avoid grass all day, just to randomly miss from there to there, trying get some fc from some sickco beatmaps, the difficulty is much different from before long times ago, and now they still trying to put many uncomfortable placements that the same star is harder than before. The star rating is a joke too, along with many plobems exist from long times ago.

And that FC indicator, I never see why it should be a stamp, just fc use great color(blue) and slider end loss use ok color or use good and ok from mania. I never see why should this ploblem be complicated.
